sftp/2.1/modules/ROOT/pages/sftp-connector-upgrade-guide.adoc

```diff
@@ -154,14 +154,47 @@ For details, review the following table:
 | Not supported. |`cc20p1305_openssh`
 |===
 
+[[supported-cryptographic-entities]]
+== Other Supported Cryptographic Entities
+
+[%header]
+|===
+|Kex Algorithms|Host Key Algorithms|MACs
+|`curve25519-sha256`*|`ecdsa-sha2-nistp256-cert-v01@openssh.com`|`hmac-sha2-256-etm@openssh.com`
+|`curve25519-sha256@libssh.org`*|`ecdsa-sha2-nistp384-cert-v01@openssh.com`|`hmac-sha2-512-etm@openssh.com`
+|`curve448-sha512`*|`ecdsa-sha2-nistp521-cert-v01@openssh.com`|`hmac-sha1-etm@openssh.com`
+|`ecdh-sha2-nistp521`|`ssh-ed25519-cert-v01@openssh.com`|`hmac-sha2-256`
+|`ecdh-sha2-nistp384`|`rsa-sha2-512-cert-v01@openssh.com`|`hmac-sha2-512`
+|`ecdh-sha2-nistp256`|`rsa-sha2-256-cert-v01@openssh.com`|`hmac-sha1`
+|`diffie-hellman-group-exchange-sha256`|`ecdsa-sha2-nistp256`|`hmac-md5` (Deprecated)
+|`diffie-hellman-group18-sha512`|`ecdsa-sha2-nistp384`|`hmac-md5-96` (Deprecated)
+|`diffie-hellman-group17-sha512`|`ecdsa-sha2-nistp521`|`hmac-sha1-96` (Deprecated)
+|`diffie-hellman-group16-sha512`|`ssh-ed25519`**|
+|`diffie-hellman-group15-sha512`|`sk-ecdsa-sha2-nistp256@openssh.com`|
+|`diffie-hellman-group14-sha256`|`sk-ssh-ed25519@openssh.com`**|
+|`diffie-hellman-group1-sha1` (Deprecated)|`rsa-sha2-512`|
+|`diffie-hellman-group14-sha1` (Deprecated)|`rsa-sha2-256`|
+|`diffie-hellman-group-exchange-sha1` (Deprecated)|`ssh-rsa`|
+||`ssh-dss` (Deprecated)|
+||`ssh-dss-cert-v01@openssh.com` (Deprecated)|
+||`ssh-rsa-cert-v01@openssh.com` (Deprecated)|
+|===
+
+[NOTE]
+* These algorithms require Bouncy Castle on Java versions before Java 11.
+* Provide externally EdDSA (Edwards-curve Digital Signature Algorithm)library.
```

Comment:
Actually, double * was deliberately added as in the above table, there are few algos which have * after them and some have ** after them. So, just wanted to ensure that point highlighted with single * inside NOTE corresponds to those algos which have single * only.

If you have any other way to visualize, please check.